Activities/Cryptography/UiPath.Cryptography.Activities/NetCore/ViewModels/DecryptFileViewModel.cs

```diff
@@ -164,5 +200,31 @@ private void KeyInputModeChanged_Action()
                     throw new NotImplementedException();
             }
         }
+
+        /// <summary>
+        /// File input Mode has changed. Set controls visibility based on selection
+        /// </summary>
+        private void FileInputModeChanged_Action()
+        {
+            switch (FileInputModeSwitch.Value)
+            {
+                case FileInputMode.File:
```

Comment:
Suggestion. Do a ResetAll() before the switch, in the switch just set the visible and required flags for those properties that qualify.

Reply:
done